### DISCUSSION TITLE
Looking for people to help me implement Nteract (Hydrogen) Jupyter notebook

### DISCUSSION THREAD

**Discussion by Tom-Boscher** [zed-industries/zed#7879] [Feature Requests]
As discuss here https://github.com/zed-industries/zed/issues/5273, I'm looking for people to start working on a integration of [nteract](https://github.com/nteract/hydrogen) Jupyter but I'm a really Rust beginner.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
